go/src/koding/klient/deploy-entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
You dev instance - if you decide we need less/more entrypoints, you'd run it from your dev instance provided you have creds for prod S3 configured.

I've created it to not manually upload script files, as I'm lazy. Most probably no-one will never run it again. Adding just in case.